homeassistant/components/tibber/sensor.py

```diff
@@ -438,6 +439,61 @@ async def _fetch_data(self) -> None:
         ]["estimatedAnnualConsumption"]
 
 
+class TibberSensorElPriceExclTax(TibberSensor):
+    """Representation of a Tibber sensor for el price excluding tax."""
+
+    _attr_state_class = SensorStateClass.MEASUREMENT
+    _attr_translation_key = "electricity_price_excl_tax"
+
+    def __init__(self, tibber_home: tibber.TibberHome) -> None:
+        """Initialize the sensor."""
+        super().__init__(tibber_home=tibber_home)
+        self._last_updated: datetime.datetime | None = None
+        self._spread_load_constant = randrange(TWENTY_MINUTES)
+
+        self._attr_available = False
+        self._attr_icon = ICON
```

Comment:
Side note: Nowadays we set the icon in `icons.json` using a translation key.

https://developers.home-assistant.io/docs/core/entity#icon-translations